From 3c19e7bc61701b46c76e2c02076a4ddfd345ddb0 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Thu, 10 Aug 2023 14:59:43 -0700
Subject: [PATCH] Handle alternate pdmp file location for x11

---
 src/emacs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/emacs.c b/src/emacs.c
index e63b092..5ddba05 100644
--- a/src/emacs.c
+++ b/src/emacs.c
@@ -871,7 +871,7 @@ load_pdump (int argc, char **argv)
 #ifdef NS_SELF_CONTAINED
     "Emacs"
 #else
-    "emacs"
+    "emacs-x11"
 #endif
     ;
 
-- 
2.41.0

